nix-parser/src/ast/macros.rs

```diff
@@ -276,6 +276,9 @@ macro_rules! unary {
 #[doc(hidden)]
 #[macro_export]
 macro_rules! atomic {
+    (antiquote!($antiquote:expr)) => {
```

Comment:
Since Rust macros are not powerful enough, here we allow users to compose some AST nodes manually at `atomic` positions